addons/escoria-core/game/core-scripts/esc/commands/walk_to_pos.gd

```diff
@@ -19,8 +21,8 @@ class_name WalkToPosCommand
 func configure() -> ESCCommandArgumentDescriptor:
 	return ESCCommandArgumentDescriptor.new(
 		3,
-		[TYPE_STRING, TYPE_INT, TYPE_INT],
-		[null, null, null]
+		[TYPE_STRING, TYPE_INT, TYPE_INT, TYPE_BOOL],
+		[null, null, null, null]
```

Comment:
Use "false" as the default value instead of null

Reply:
Done